README.md

```diff
@@ -64,6 +71,27 @@ carthage update
 ```
 In your application targets “General” tab under the “Linked Frameworks and Libraries” section, drag and drop lottie-ios.framework from the Carthage/Build/iOS directory that `carthage update` produced.
 
+### Binary dependency (`xcframework`)
+
+Lottie is also available as a `xcframework` binary dependency. Lottie's git repository is somewhat large (300+ MB), so using an `xcframework` (~10MB) reduces the amount of data that needs to be downloaded.
+
+Since Lottie 4.0.1, a `Lottie.xcframework.zip` archive is attached to each release. You can find the latest release [here](https://github.com/airbnb/lottie-ios/releases/latest). 
```

Comment:
Where should folks look to find the checksum?

Reply:
Not sure the best way to handle this. I image you'd have to download the artifact and run a command like `swift package compute-checksum Lottie.xcframework.zip`. I can add a note about this to the readme.

Reply:
Maybe we could include it somewhere in the release notes?

Reply:
Good idea, we'll include the checksum in the release notes moving forward